text/0193-fixing-type-unions.md

```diff
@@ -0,0 +1,558 @@
+---
+feature name: fixing-type-unions
+start date: 2020-07-06
+rfc pr: 194
+related issue: 193
+---
+
+# Summary
+
+_Type unions_ is a feature of the TypeScript language that allows typing values as "one of several candidate types".
+This is achieved by specifying a `|`-delimited list of candidate types (e.g: `Foo | Bar`). While this feature is common
+in dynamic languages (**TypeScript**, **Python**, ...), it is almost always absent from statically typed languages
+(**Java**, **C#**, ...). In those languages, the impossibility to represent _type unions_ natively results in degraded
+developer experience due to the reduced fidelity of the generated typings.
+
+This RFC proposes to introduce new features in the _jsii_ compiler to support generation of additional code that will
+allow statically typed languages to benefit from the flexibility of type unions without having to endure the experience
+degradations they are currently exposed to.
+
+# Release Notes
```

Comment:
This is the jsii RELEASE NOTES, please also add the CDK's "BREAKING CHANGE" section for 2.x which describes for e.g. .NET and Java users what's changed for them (before/after)